spec/cnb_spec.rb

```diff
@@ -1,23 +1,7 @@
 require 'spec_helper'
 
 describe 'CNB' do
-  describe '#daily_rates' do
-    before(:each) do
-      CNB::DailyRates.any_instance.stub(:get_currencies) { {} }
-    end
-
-    it 'returns DailyRates object' do
-      expect(CNB.daily_rates).to be_a(CNB::DailyRates)
-    end
-  end
-
-  describe '#monthly_rates' do
-    before(:each) do
-      CNB::MonthlyRates.any_instance.stub(:get_currencies) { {} }
-    end
-
-    it 'returns MonthlyRates object' do
-      expect(CNB.monthly_rates(11,2013)).to be_a(CNB::MonthlyRates)
-    end
+  it 'has a version number' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.